stubs/resources/views/chart/script.blade.php

```diff
@@ -33,6 +33,18 @@
         xaxis: {
             categories: {!! $chart->xAxis() !!}
         },
+        @if ($chart->yAxisMin() !== null && $chart->yAxisMax() !== null && $chart->yAxisTickAmount() !== null)
```

Comment:
I would avoid too complex if statements, I would prefer to handle every property with it's own if statement or using a ternary operator

Reply:
after using Livewire 3 the if-statement doesnt work, so I removed it